### ISSUE TITLE
May I know why need to use the facts.zip

### ISSUE THREAD

**Issue by lihuiliullh** [nddsg/KGMiner#30]
May I know why need to use the facts? Looking forward to your reply.

wget "https://www.dropbox.com/s/uezdxht4elob6b3/facts.zip?dl=1" -O facts.zip

**Comment by bxshi** [comment]
I believe they are processed input data used to conduct the experiments
mentioned in the paper.

**Comment by lihuiliullh** [comment]
You mean they are the training data? So, the files outside of the facts dir are testing data?

**Comment by bxshi** [comment]
Could you download them and verify? I think they might be exactly the same
as the ones in the facts folder. Sorry don’t have a computer next to me
right now.

**Comment by lihuiliullh** [comment]
Yes, I download them. Some of them are the same except that there are several xxx.simrank.csv files in the facts folder. That's ok. Thanks for your help.
